dp/jobqueueing/gcreset.py

```diff
@@ -0,0 +1,96 @@
+"""
+Script to update the status of a queue entry.
```

Comment:
Apologies. This was developed pretty quickly, and I actually changed my mind about the workflow partway through (from emails to output of qstat). I'll make some major improvements to the documentation.